### PR TITLE
Register ConicIP: v0.1.0, v0.1.1

### PR DESCRIPTION
**Pull request by gabgoh**
https://github.com/JuliaLang/METADATA.jl/pull/5839

third time's the charm, hopefully

### CODE REVIEW

**Comment by mlubin** [comment]
@tkelman, okay to merge this?

File: ConicIP/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4+
+MathProgBase
```
Comment (tkelman) [inline ConicIP/versions/0.1.0/requires]: not having an upper bound here is likely to cause issues when MathProgBase 0.6 happens

File: ConicIP/versions/0.1.1/requires [hunk 2]
```diff
@@ -0,0 +1,4 @@
+julia 0.4 
+MathProgBase 0.5 0.6
+WoodburyMatrices
+Compat
```
Comment (tkelman) [inline ConicIP/versions/0.1.1/requires]: should declare a minimum version dependency that implements `Compat.view` (or any more recent functionality you need)

File: ConicIP/versions/0.1.0/requires (hunk 1 above)
Comment (mlubin) [inline ConicIP/versions/0.1.0/requires]: Probably best to just drop the 0.1.0 tag?

Comment (tkelman) [inline ConicIP/versions/0.1.0/requires]: @gabgoh, Miles was referring to here in metadata. It looks like you deleted the git tags from https://github.com/MPF-Optimization-Laboratory/ConicIP.jl/releases, so let's re-do this again. Add a minimum version requirement on Compat (from a quick glance it didn't look like you were using anything more recent than `Compat.view`) then we can re-tag and merge.

**Pull request closed without merging**